deploy/charts/trust-manager/values.yaml

```
@@ -101,6 +101,18 @@ app:
     # -- If false, disables the default seccomp profile, which might be required to run on certain platforms
     seccompProfileEnabled: true
 
+secretTargets:
+  # -- If set to true, enable writing trust bundles to Kubernetes Secrets as a target
+  enabled: false
+  # -- If set to true, grant read/write permission to all secrets. Use it with CAUTIOUS!
+  #    When set, ignores authorizedSecrets list setting.
```

nitpick: I think this has a typo, and I'd like to suggest a change.

```suggestion
  # -- If set to true, grant read/write permission to all secrets across the cluster. Use with caution!
  #    When set, ignores authorizedSecrets list setting.
```

---

Done